projects/core/src/lib/plugins/change-event-plugin.ts

```diff
@@ -0,0 +1,33 @@
+import type {MaskitoPlugin} from '../types';
+
+export function maskitoChangeEventPlugin(): MaskitoPlugin {
+    return element => {
+        if (element.isContentEditable) {
+            return;
+        }
+
+        let value = element.value;
+
+        const focusListener = (): void => {
+            value = element.value;
+        };
+        const changeListener = (): void => {
+            value = element.value;
+        };
+        const blurListener = (): void => {
+            if (element.value !== value) {
+                element.dispatchEvent(new Event('change'));
```

Comment:
Change event bubbles, need to add `{ bubbles: true }`

Reply:
Fixed in https://github.com/taiga-family/maskito/pull/1338/commits/55192a78648be4d4c17068c53347d8ca5af5115a